src/components/Textarea/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+interface Textarea {
+  readonly name: string
+  readonly label: string
+  readonly placeholder?: string
+  readonly errorMessage?: string
+  readonly onChange: (event: React.ChangeEvent<HTMLTextAreaElement>) => void
+  readonly textareaRef: (ref: HTMLTextAreaElement) => void
+}
+
+const Textarea = ({ name, label, placeholder, errorMessage, onChange, textareaRef }: Textarea) => {
```

Comment:
Is this intention to avoid duplicate name? If not then should it be this?
```suggestion
const TextArea = ({ name, label, placeholder, errorMessage, onChange, textareaRef }: Textarea) => {
```

Reply:
My intention was to follow what Google's product do. In Material UI, they use the word `textarea` for naming their custom component. What do you think

![image](https://user-images.githubusercontent.com/32285706/115950067-4a010d80-a503-11eb-93d4-3edfa5bd39e1.png)